py/README.md

```diff
@@ -0,0 +1,374 @@
+at least one of the filters in the list, and `not` to match results that do NOT match
+the filter it is placed on.
+
+##### Result filters
+
+A [ResultFilters object](swagger_docs/ResultFilters.md) specifies criteria that
+table query results must match to be returned. If no results match the criteria, no 
+results will be returned.
+
+Exactly one of `column_filter`, `all_of`, or `any_of` must be specified on each
+ResultFilters.
+
+You can construct arbitrarily complex matching criteria by using nested
+`ResultFilters` with `all_of` or `any_of`.
+
+##### ResultFilters fields
+
+Name | Required? | Description
+---- | --------- | -----------
+column_filter | No | A [ColumnFilter](#column-filters) specifying that a column value in a table in the CDR must match a provided value. 
+all_of | No | A list of [ResultFilters](#result-filters) specifying a list of conditions that must ALL be satisfied for results that are returned.
+any_of | No | A list of [ResultFilters](#result-filters) specifying a list of conditions that at least one of must be satisfied for results that are returned.
+if_not | No | If set to True, only results that DO NOT match the criteria specified by this filter are returned. Defaults to False.
+
+##### Column filters
+
+A [ColumnFilter object](swagger_docs/ColumnFilters.md) specifies a filter
+that a given column value in a table in the CDR must match a provided value.
+
+If the `IN` operator is used, exactly one of `values` and `value_numbers` must be specified,
+depending on the type of column being compared against.
+
+If the `LIKE` operator is used, only `value` should be specified.
+
+If any other operator is used (or no operator is specified), 
+exactly one of `value`, `value_number`, `value_date`, and `value_null` must be specified,
+depending on the type of column being compared against.
+ 
+
+##### ColumnFilter fields
+
+Name | Required? | Description
+---- | --------- | -----------
+column_name | Yes | The name of the column on the primary table or [related table column](#related-table-columns) indicating what column to match against.
+operator | No | The [Operator](swagger_docs/Operator.md) to use in the comparison. Defaults to `EQUAL`.
+value | No | A string value to use in comparisons with string columns.
+values | No | A list of string values to use in comparisons with string columns using the `IN` operator.
+value_number | No | A numeric value to use in comparisons with numeric columns.
+value_numbers | No | A list of numeric values to use in comparisons with numeric columns using the `IN` operator.   
+value_date | No | A date value to use in comparisons with date columns.
+value_null | No | Set to true if you wish to compare to NULL / not set value (with the `EQUAL` operator for checking for matching NULL, or the `NOT_EQUAL` operator for matching anything but NULL.)  
+
+##### TableQuery examples
+
+All examples below reference modules that can be imported from `aou_workbench_client.swagger_client.models`.
```

Comment:
Future idea: generating these from the Python docs and then testing via [doctest](https://docs.python.org/2/library/doctest.html) would be pretty cool.